### PR TITLE
prime_number.py Added

### PR DESCRIPTION
**Pull request by charlie219**
Added prime_number.py under math folder of Python. The added file is equipped to check the primality of the passed argument 
in both, iterative as well as recursive approach.

I have also changed the `CONTRIBUTORS.md` file and added my name into it with all the required links.

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Y

### Description

Solved issue #1372 in python as assigned by @iamrajiv 
the task was to make a function to check the primality of a number (both Iterative and Recursive method).
I solved the issue with python

### Checklist

- [Y] I've read the contribution guidelines.
- [Y] I've checked the issue list before deciding what to submit.
- [N] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#1372 #1411 #1410

### CODE REVIEW

File: Python/math/prime_number.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    #check the next divisor
+    return is_prime_recursive(num,i+1)
+
+#-------------------------------------------------Driver Code---------------------------------------------------
+
+if __name__ == "__main__":
+    n = int(input("Enter the number to be checked : "))
+    print("Iterative Approch:- ",is_prime_iterative(n))
+    print("Recursive Approch:- ",is_prime_recursive(n))
+
```
Comment (atarax665) [inline Python/math/prime_number.py]: Add sample input/output for your code.

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -164,6 +164,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
     <td align="center"><a href="https://github.com/Gaurav6982"><img src="https://avatars0.githubusercontent.com/u/49642550?v=4" width="100px;" alt=""/><br /><sub><b>Gaurav</b></sub></a><br /><a href="https://github.com/TesseractCoding/NeoAlgo/commits?author=Gaurav6982" title="Code">💻</a></td>
     <td align="center"><a href="http://techsfortalk.me"><img src="https://avatars3.githubusercontent.com/u/64386187?v=4" width="100px;" alt=""/><br /><sub><b>Ashutosh_K_Sah</b></sub></a><br /><a href="https://github.com/TesseractCoding/NeoAlgo/commits?author=ashu-cybertron" title="Code">💻</a></td>
+    <td align="center"><a href="https://github.com/charlie219"><img src="https://avatars2.githubusercontent.com/u/43098511?v=4" width="100px;" alt=""/><br /><sub><b>Akash Kumar Bhagat</b></sub></a><br /><a href="https://github.com/TesseractCoding/NeoAlgo/commits?author=charlie219" title="Code">💻</a></td>
```
Comment (atarax665) [inline CONTRIBUTORS.md]: Yo don't need to update ```CONTRIBUTORS.md``` manually, you will be added as contributor once your code is merged.

**Review by atarax665 — CHANGES_REQUESTED** [review]
Update README.md.

File: CONTRIBUTORS.md (hunk 2 above)
Comment (charlie219) [reply CONTRIBUTORS.md]: okay then I will revert the changes done to `CONTRIBUTORS.md` and add sample Input and Output

**Comment by charlie219** [comment]
@atarax665 All the desired changes are made, please review the PR

File: Python/math/prime_number.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
+'''
+    is_prime_iterative() function uses an iterative approch to chech the primality
```
Comment (atarax665) [inline Python/math/prime_number.py]:
```suggestion
    is_prime_iterative() function uses an iterative approach to check the primality
```

File: Python/math/prime_number.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+of the pased argument.
```
Comment (atarax665) [inline Python/math/prime_number.py]:
```suggestion
of the passed argument.
```

File: Python/math/prime_number.py [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    It checks for each number i from 2 to sqrt(num)+1 that wether it divides the
```
Comment (atarax665) [inline Python/math/prime_number.py]:
```suggestion
    It checks for each number i from 2 to sqrt(num)+1 that whether it divides the
```

File: Python/math/prime_number.py [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+passed number or not. If it does the passed number is not prime, otherwise the
```
Comment (atarax665) [inline Python/math/prime_number.py]:
```suggestion
passed number or not. If it does, the passed number is not prime, otherwise the 
```

**Review by atarax665 — CHANGES_REQUESTED** [review]
Format your code according to PEP8 rules. You can [this](http://pep8online.com/).

**Comment by charlie219** [comment]
@atarax665 the code is formatted to pep8 style and the typos are corrected.

**Review by atarax665 — APPROVED** [review]

**Comment by charlie219** [comment]
@plazzy99 please review the PR and mereg it

**Review by iamrajiv — APPROVED** [review]

**Comment by atarax665** [comment]
@all-contributors please add @charlie219  for code

**Pull request merged**